specification/compute/resource-manager/Microsoft.Compute/stable/2021-08-01/disk.json

```diff
@@ -2814,6 +2842,14 @@
               {
                 "value": "CopyStart",
                 "description": "Create a new disk by using a deep copy process, where the resource creation is considered complete only after all data has been copied from the source."
+              },
+              {
+                "value": "ImportSecure",
+                "description": "Similar to Import create option. Create a new TrustedLaunch or ConfidentialVM supported disk by importing additional VM guest state blob specified by securityDataUri in storage account specified by storageAccountId"
+              },
+              {
+                "value": "UploadPreparedSecure",
+                "description": "Similar to Upload create option. Create a new TrustedLaunch or ConfidentialVM supported disk and upload using write token in both disk and VM guest state blob"
```

Comment:
"Similar to Upload create option. Create a new TrustedLaunch or ConfidentialVM supported disk and upload using write token in both disk and VM guest state blob"
should be modified as
"Similar to Upload create option. Create a new Trusted Launch VM or Confidential VM supported disk and upload using write SAS token to both disk and VM guest state blob"

Reply:
@anshulsolanki21